### PR TITLE
General Edits

### LINKED ISSUE #58: General review - remove forward references
Twin issues: 
- https://github.com/scionassociation/scion-dp_I-D/issues/45
- https://github.com/scionassociation/scion-cppki_I-D/issues/36

- [x] @knmeynell update terminology and de-hyphenate as in https://github.com/scionassociation/scion-dp_I-D/pull/51
- [x] @nicorusti general re-read through the draft

### PR DESCRIPTION
**Pull request by knmeynell**
General edits to improve consistency, readability and to rationalise superfluous text. Also fixed some unclear or confusing text.

Resolves #58

### CODE REVIEW

File: draft-dekater-scion-controlplane.md [hunk 1]
```diff
@@ -1262,162 +1267,165 @@ The last step of the above described core and intra-ISD propagation procedures i
...
 
-As mentioned previously, a non-core AS typically receives several PCBs representing several path segments to the core ASes of the ISD the AS belongs to. Out of these PCBs, the non-core AS selects those down-path segments through which it wants to be reached, based on AS-specific selection criteria. The next step is to register the selected down-segments with the control service of the relevant core ASes, according to a process called *intra-ISD path-segment registration*. As a result, a core AS's control service contains all intra-ISD path segments registered by the non-core ASes of its ISD. In addition, each core AS control service also stores preferred core-path segments to other core ASes, in the *core-segment registration* process. Both processes are described below.
+As mentioned previously, a non-core AS typically receives several PCBs representing several path segments to the core ASes of the ISD the AS belongs to. Out of these PCBs, the non-core AS selects those down path segments through which it wants to be reached, based on AS-specific selection criteria.
```
Comment (jiceatscion) [inline draft-dekater-scion-controlplane.md:1350]: "As mentioned previously, "

Do we need to apologize for it?

File: draft-dekater-scion-controlplane.md [hunk 2]
```diff
@@ -1262,162 +1267,165 @@ The last step of the above described core and intra-ISD propagation procedures i
...
 
-The up- and down-segments do not have to be equal. An AS may want to communicate with core ASes via one or more up-segments that differ from the down-segment(s) through which it wants to be reached. Therefore, an AS can define different selection policies for the up- and down-segment sets. Also, the processes of transforming a PCB in an up-segment or a down-segment differ slightly. Both processes are described below.
+Every *registration period* (determined by each AS), the AS's Control Service selects two sets of PCBs to transform into two types of path segments:
+
+- Up segments, which allow the infrastructure entities and endpoints in this AS to communicate with core ASes; and
+- Down segments, which allow remote entities to reach this AS.
+
+The up segments and down segments do not have to be equal as AS may want to communicate with core ASes via one or more up segments that differ from the down segment(s) through which it wants to be reached. Therefore, an AS can define different selection policies for the up segment and down segment sets. In addition, the processes of transforming a PCB in an up segment or a down segment differ slightly.
```
Comment (jiceatscion) [inline draft-dekater-scion-controlplane.md:1364]: "as AS may want to" -> "as an AS may want to"

File: draft-dekater-scion-controlplane.md [hunk 3]
```diff
@@ -144,37 +144,37 @@ The SCION architecture was initially developed outside of the IETF by ETH Zurich
...
+**Forwarding Path**: A forwarding path is a complete end-to-end path between two SCION endpoints which is used to transmit packets in the data plane. It can be created with a combination of up to three path segments (an up segment, a core segment, and a down segment).
 
-**Hop Field (HF)**: As they traverse the network, path-segment construction beacons (PCBs) accumulate cryptographically protected AS-level path information in the form of hop fields. In the data plane, hop fields are used for packet forwarding: they contain the incoming and outgoing interface IDs of the ASes on the forwarding path.
+**Hop Field (HF)**: Hop Field (HF): As they traverse the network, Path Segment Construction Neacons (PCBs) accumulate cryptographically protected AS-level path information in the form of Hop Fields. In the data plane, Hop Fields are used for packet forwarding: they contain the incoming and outgoing interface IDs of the ASes on the forwarding path.
```
[the remark this answers is not in the document]
Comment (knmeynell) [reply draft-dekater-scion-controlplane.md:161]: Typo in 'Path Segment Construction Beacons (PCBs)

File: draft-dekater-scion-controlplane.md [hunk 4]
```diff
@@ -144,37 +144,37 @@ The SCION architecture was initially developed outside of the IETF by ETH Zurich
...
 
-**Info Field (INF)**: Each path-segment construction beacon (PCB) contains a single info field, which provides basic information about the PCB. Together with hop fields (HFs), info fields are used to create forwarding paths.
+**Info Field (INF)**: Info Field (INF): Each Path Segment Construction Beacon (PCB) contains a single Info field, which provides basic information about the PCB. Together with Hop Fields (HFs), these are used to create forwarding paths.
 
-**Isolation Domain (ISD)**: In SCION, Autonomous Systems (ASes) are organized into logical groups called isolation domains or ISDs. Each ISD consists of ASes that span an area with a uniform trust environment (i.e., a common jurisdiction). A possible model is for ISDs to be formed along national boundaries or federations of nations.
+**Isolation Domain (ISD)**: Isolation Domain (ISD): In SCION, Autonomous Systems (ASes) are organized into logical groups called Isolation Domains or ISDs. Each ISD consists of ASes that span an area with a uniform trust environment (e.g. a common jurisdiction). A possible model is for ISDs to be formed along national boundaries or federations of nations.
```
[the remark this answers is not in the document]
Comment (knmeynell) [reply draft-dekater-scion-controlplane.md:165]: Okay

File: draft-dekater-scion-controlplane.md [hunk 5]
```diff
@@ -1050,14 +1057,14 @@ The following code block defines the peer entry component `PeerEntry` in Protobu
 
 ### PCB Extensions {#pcb-ext}
 
-In addition to basic routing information like hop entries and peer entries, PCBs can be used to communicate additional metadata, in its extensions. Extensions can be signed and unsigned. Signed extensions are protected by the AS signature, whereas unsigned extensions are not.
+In addition to basic routing information such a hop entries and peer entries, PCBs can be used to communicate additional metadata in extensions. Extensions can be signed and unsigned: signed extensions are protected by the AS signature, whereas unsigned extensions are not.
```
[the remark this answers is not in the document]
Comment (knmeynell) [reply draft-dekater-scion-controlplane.md:1058]: > "such a" -> "such as"

Okay

File: draft-dekater-scion-controlplane.md [hunk 6]
```diff
@@ -1262,162 +1267,165 @@ The last step of the above described core and intra-ISD propagation procedures i
...
 
-This bias comes in addition to a structural delay: PCBs are propagated at a configurable interval (typically, around one minute). As a result of this and the way they are iteratively constructed, PCBs with N hops may be validated up to N intervals (so maximally N minutes) after origination. This creates a constraint on the expiration of hops. Hops of the minimal expiration time (337.5 seconds - see [](#hopfield)) would render useless any PCB describing a path longer than 5 hops. For this reason, it is unadvisable to create hops with a short expiration time, that should be around 6 hours.
+This bias comes in addition to a structural delay: PCBs are propagated at a configurable interval - typically around one minute. As a result of this and the way they are iteratively constructed, PCBs with N hops may be validated up to N intervals (so maximally N minutes) after origination which creates a constraint on the expiration of hops. Hops of the minimal expiration time (337.5 seconds - see [](#hopfield)) would render useless any PCB describing a path longer than 5 hops and for this reason it is unadvisable to create hops with a short expiration time - they SHOULD be around 6 hours.
```
[the remark this answers is not in the document]
Comment (knmeynell) [reply draft-dekater-scion-controlplane.md:1282]: Fine with me.

File: draft-dekater-scion-controlplane.md [hunk 7]
```diff
@@ -1262,162 +1267,165 @@ The last step of the above described core and intra-ISD propagation procedures i
...
-Ultimately, the number of PCBs received by an AS per propagation interval remains bounded by 50 for each parent link of an AS, and at most 50 PCBs per child link are propagated. The length of these PCBs, and thus the number of AS entries to be processed and stored, is expected to be moderate and not grow considerably with network size. The total resource overhead for beacon propagation is easily manageable even for highly connected ASes.
+This typically produces an acyclic graph which is narrow at the top, widens towards the leafs, and is relatively shallow. Intermediate provider ASes will have a large number of children, while they only have a small number of parents and the chain of intermediate providers from a leaf AS to a core AS is typically not long (e.g. local, regional, national provider, then core).
```
[the remark this answers is not in the document]
Comment (knmeynell) [reply draft-dekater-scion-controlplane.md:1313]: Okay

File: draft-dekater-scion-controlplane.md [hunk 8]
```diff
@@ -1262,162 +1267,165 @@ The last step of the above described core and intra-ISD propagation procedures i
...
-If the same AS has 1000 child links, the propagation of the beacons will require signing one new AS entry for each of the propagated PCBs for each link (at most 50 per link), that is at most 50000 signatures per propagation event.
-The total bandwidth for the propagation of these PCBs for all 1000 child links would, again very roughly, be around 25MB/s.
-All of these are manageable with even modest consumer hardware.
+Each AS potentially receives PCBs for all down path segments from the core to itself. While the number of distinct provider chains to the core is typically moderate, the multiplicity of links between provider ASes has multiplicative effect on the number of PCBs. Once this number grows above the maximum recommended best PCBs set size of 50, ASes SHOULD trim the set of PCBs propagated.
```
[the remark this answers is not in the document]
Comment (knmeynell) [reply draft-dekater-scion-controlplane.md:1315]: Either is grammatically correct, although J.C's suggestion is probably better.

File: draft-dekater-scion-controlplane.md (hunk 1 above)
Comment (knmeynell) [reply draft-dekater-scion-controlplane.md:1348]: Okay

File: draft-dekater-scion-controlplane.md (hunk 2 above)
Comment (knmeynell) [reply draft-dekater-scion-controlplane.md:1362]: Okay

File: draft-dekater-scion-controlplane.md [hunk 9]
```diff
@@ -144,37 +144,37 @@ The SCION architecture was initially developed outside of the IETF by ETH Zurich
...
 
-**Hop Field (HF)**: As they traverse the network, path-segment construction beacons (PCBs) accumulate cryptographically protected AS-level path information in the form of hop fields. In the data plane, hop fields are used for packet forwarding: they contain the incoming and outgoing interface IDs of the ASes on the forwarding path.
+**Hop Field (HF)**: Hop Field (HF): As they traverse the network, Path Segment Construction Neacons (PCBs) accumulate cryptographically protected AS-level path information in the form of Hop Fields. In the data plane, Hop Fields are used for packet forwarding: they contain the incoming and outgoing interface IDs of the ASes on the forwarding path.
 
-**Info Field (INF)**: Each path-segment construction beacon (PCB) contains a single info field, which provides basic information about the PCB. Together with hop fields (HFs), info fields are used to create forwarding paths.
+**Info Field (INF)**: Info Field (INF): Each Path Segment Construction Beacon (PCB) contains a single Info field, which provides basic information about the PCB. Together with Hop Fields (HFs), these are used to create forwarding paths.
```
[the remark this answers is not in the document]
Comment (knmeynell) [reply draft-dekater-scion-controlplane.md:163]: Okay

**Review by nicorusti — COMMENTED** [review]
Commenting just to resolve some conversations.. I am reviewing

**Review by nicorusti — APPROVED** [review]

**Pull request merged**